lib/main.dart

```diff
@@ -84,6 +85,7 @@ class MyHomePage extends StatelessWidget {
     final appState = context.watch<MyAppState>();
 
     // Building the app's UI
+    //Majd
```

Comment:
you could refactor the build method by creating separate widgets/methods for different screen size layouts for better readability and maintainability.